packages/@aws-cdk/triggers/README.md

```diff
@@ -0,0 +1,97 @@
+# Triggers
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![cdk-constructs: Stable](https://img.shields.io/badge/cdk--constructs-stable-success.svg?style=for-the-badge)
+
+---
+
+<!--END STABILITY BANNER-->
+
+
+Execute code as part of a CDK app deployment.
+
+## Usage
+
+You can trigger the execution of an AWS Lambda function during deployment after
+a resource or groups of resources are provisioned.
+
+The library includes constructs that represent different triggers. The `BeforeCreate` and `AfterCreate` constructs can be used to trigger a handler before/after a set of resources have been created.
+
+```ts
+import { Trigger } from '@aws-cdk/triggers';
+
+new Trigger(this, 'MyTrigger', {
```

Comment:
This is neither `BeforeCreate` nor `AfterCreate`. I like those names better, actually.

Also, the code doesn't seem to support the `BeforeCreate` case?

Reply:
OK, now we have support for `executeAfter` and `executeBefore`. See updated README.